src/Model/Table/ContributionsTable.php

```diff
@@ -321,9 +318,8 @@ public function excludeBots($query)
     public function getLastContributionOf($userId)
     {
         return $this->find()
-            ->select(['ip', 'count' => 'count(*)'])
+            ->select(['count' => 'count(*)'])
             ->where(['user_id' => $userId])
-            ->group(['group' => 'ip'])
             ->orderDesc('count')
             ->limit(10)
             ->all();
```

Comment:
This method is not used anywhere else, so you could as well remove it altogether.

Reply:
Alright, done.